integrations/observability/nginx/nginx-1.0.0.json

```diff
@@ -44,6 +44,7 @@
     { "name": "logs", "version": "1.0.0" }
   ],
   "assets": [
+    { "name": "getting-started-tutorial", "version": "1.0.0", "extension": "ndjson", "type": "savedObjectBundle", "workflows": ["dashboards"] },
```

Comment:
What is the intended flow to present this to the user? Making it a saved object bundle associated with the integration means that it won't be added to the cluster until after they set up the integration.